site/content/internal/writing-a-blog-post.md

```diff
@@ -5,11 +5,56 @@ section: internal
-2. Create a new .md file in the [/site/content/blog/](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/blog) folder
-  - Use `YYYY-MM-DD-<your-blog-post-title>.md` as the filename
+Blog posts can cover a wide range of topics, such as:
+
+- Addressing a customer-facing problem
+- Describing an experience with Mattermost/your Mattermost implementation 
+- Sharing information about cool tech 
+- Sharing feedback on an interesting talk or conference 
+- Part of a Hackathon project
+- A Help Wanted ticket
+- A knowledge-share and call for feedback/community engagement
+- A discussion of a specific problem or improvement that you worked on
+- A breakdown of a new process or technology you’re using
+
+
+Once you've got the topic in mind - what it's about, what you want to achieve with the post, what the next steps are etc. it’s sometimes helpful to start writing the conclusion and expand to your jumping off point to introduce your topic/idea/discovery draws 
+
+Make a note of your intended audience, so you can decide whether to use very technical terms/jargon, or spend time unpacking terminology.
+
+
+Structuring Your Blog Post
+--------------------------
+
+These are some ideas of the parts of a blog post. They’re not mandatory and not all blog posts will include every aspect. What works for some posts won’t work for others. 
+
+- **Introduction/overview:** An opening paragraph detailing the goal of the post and the technologies/processes that were used. For example: “Monitoring is an essential part of our organization. When we started, we were using “x” which gave us insight into “y”. With our growth as a company, we need more insights into areas that “x” can’t handle, so we decided to switch to “a” and “b””.  
+- **Problem/Solution/Situation:** Detail the problem or scenario that sparked the blog post. For example, “our monitoring tools weren’t giving us the insight we needed, and were costing a lot of money. We decided to try solve this by using multiple tools that could be used or on standby as needed. This saves money.” 
+- **Environment:** Plugins, software, specific configurations/services used. 
+- **Steps:** The process followed to get from conception to implementation, for example, how the monitoring was configured initially, and the new configuration. Include code samples/screenshots. How long it took to build a database/history. What steps were taken to create a good alert system. 
+- **Samples:** Code samples are often useful, as are screenshots/animated GIFs if a complex process is being demonstrated. 
+- **Benefits:** The benefits of the exercise or process of the blog post - man-hours saved, budget target achieved, lower overheads etc. 
+- **Conclusion:** Whether the exercise or process has long-term potential, whether it’s still in place, or whether it was a failure. 
+
+Some popular blogs that are worth reading include:
```

Comment:
These are not rendering as a list

Reply:
Ah I didn't check that - I'll fix the formatting!